src/main/java/werkit/UI.java

```diff
@@ -0,0 +1,743 @@
+     */
+    protected String getUserInput() {
+        String userInput = inputReader.nextLine();
+        String userInputTrimmed = userInput.trim();
+        printLine();
+
+        return userInputTrimmed;
+    }
+
+    /**
+     * Prints a message when a new workout has been created. The newly created workout will also be displayed.
+     *
+     * @param newWorkout The Workout object that is newly created.
+     */
+    public void printNewWorkoutCreatedMessage(Workout newWorkout) {
+        System.out.println(NEW_WORKOUT_CREATED_MESSAGE);
+        System.out.println();
+        System.out.println("\t" + newWorkout.toString());
+        System.out.println();
+    }
+
+    /**
+     * Prints a message when a workout has been deleted. Deleted workout will also be displayed.
+     *
+     * @param deletedWorkout The workout to be deleted.
+     */
+    public void printDeleteWorkoutMessage(Workout deletedWorkout) {
+        System.out.println(DELETED_WORKOUT_MESSAGE);
+        System.out.println();
+        System.out.println("\t" + deletedWorkout.toString());
+        System.out.println();
+    }
+
+    /**
+     * Prints a message when a workout has been updated.
+     * Shows updated workout.
+     *
+     * @param updatedWorkout The workout that is updated.
+     */
+    public void printUpdateWorkoutMessage(String oldWorkout, Workout updatedWorkout) {
+        System.out.println(UPDATED_WORKOUT_MESSAGE + getColorText(TextColor.COLOR_YELLOW, oldWorkout) + " to:");
+        System.out.println();
+        System.out.println("\t" + updatedWorkout.toString());
+        System.out.println();
+    }
+
+    /**
+     * Prints the text in colors pre-defined in TextColor class.
+     * @param color ANSI color codes defined in TextColor class ONLY.
+     * @param text The string text that needs to be colored.
+     */
+    public void printColorText(String color, String text) {
+        System.out.println(color + text + TextColor.COLOR_RESET);
+    }
+
+    /**
+     * Formats the string to contain the ANSI color code specified.
+     *
+     * @param color The color to format the string into.
+     * @param text The text to be formatted with a color.
+     * @return The string formatted with the ANSI color code.
+     */
+    public String getColorText(String color, String text) {
+        String textWithColor = color + text + TextColor.COLOR_RESET;
+        return textWithColor;
+    }
+
+    /**
+     * Prints the help messages. To be updated.
+     */
+    public void printHelpMessage() {
+        printExerciseHelp();
+        printLine();
+        printLine();
+        printWorkoutHelp();
+        printLine();
+        printLine();
+        printPlanHelp();
+        printLine();
+        printLine();
+        printScheduleHelp();
+        printLine();
+        printLine();
+        printSearchHelp();
+        printLine();
+        printLine();
+        printExitHelp();
+    }
```

Comment:
in contrast to many other methods, this one is overdoing slap, but it is good! 😄